scylla-cdc/src/stream_generations.rs

```diff
@@ -215,7 +215,10 @@ impl GenerationFetcher {
                     match self.fetch_next_generation(&generation).await {
                         Ok(Some(generation)) => break generation,
                         Ok(None) => sleep(sleep_interval).await,
-                        _ => warn!("Failed to fetch next generation"),
+                        _ => {
+                            warn!("Failed to fetch next generation");
+                            sleep(sleep_interval).await
```

Comment:
:question: Shouldn't we introduce the same fix to other similar `_` branches as well? They suffer from the _deathloop_, too.